ToolBox/YahooDownloader/YahooDataDownloader.cs

```diff
@@ -105,7 +105,7 @@ public Queue<BaseData> DownloadSplitAndDividendData(Symbol symbol, DateTime star
                         parsed.Add(new Split
                         {
                             Time = Parse.DateTimeExact(values[0].Replace("-", string.Empty), DateFormat.EightCharacter),
-                            Value = ParseAmount(values[1])
+                            Value = 1/ParseAmount(values[1])
```

Comment:
In present form it's assigning whole value. eg. for 1 to 7 split. It's assigning 7 instead of 1/7